.travis.yml

```diff
@@ -129,6 +153,26 @@ matrix:
     before_install:
     - wget https://github.com/jgm/pandoc/releases/download/2.1.1/pandoc-2.1.1-1-amd64.deb
     - sudo dpkg -i pandoc-2.1.1-1-amd64.deb
+#   - script: scripts/test-gtapi-mock-drv.sh
+#     install:
+#     - pip install --user pybind11 nose2
+#     cache: ccache
+#     addons:
+#       apt:
+#         packages:
+#         - libjson-c-dev
+#         - libhwloc-dev
+#         - uuid-dev
+#         - doxygen
+#   - script: scripts/build-tests-default.sh
+#     cache: ccache
+#     addons:
+#       apt:
+#         packages:
+#         - libjson-c-dev
+#         - libhwloc-dev
+#         - uuid-dev
+#         - doxygen
```

Comment:
Should we get rid of this?

Reply:
@nakulkorde?

Reply:
yes I would agree. Also, the test-build script which is supposed to run three different types with different build types is being run with RelWithDebInfo in all three. I will fix that and add to this PR in a few minutes.